api/src/telemetry/lib/get-report.ts

```diff
@@ -34,17 +43,23 @@ export const getReport = async (): Promise<TelemetryReport> => {
 		database: getDatabaseClient(),
 
 		dashboards: basicCounts.directus_dashboards,
-		extensions: basicCounts.directus_extensions,
 		files: basicCounts.directus_files,
 		flows: basicCounts.directus_flows,
 		roles: basicCounts.directus_roles,
 		shares: basicCounts.directus_shares,
 
-		admin_users: userCounts.admin,
-		app_users: userCounts.app,
-		api_users: userCounts.api,
+		users_admin: userCounts.admin,
+		users_app: userCounts.app,
+		users_api: userCounts.api,
 
 		collections: userItemCount.collections,
 		items: userItemCount.items,
+
+		fields_max: fieldsCounts.max,
+		fields_total: fieldsCounts.total,
+
+		extensions: extensionsCounts.activeTotal - extensionsCounts.activeBundles,
```

Comment:
Like mentioned in the comment for `get-extension-count.ts` we should do all the work inside of the function because `activeTotal` currently has an ambiguous meaning.